wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSessionBuilder.swift

```diff
@@ -205,7 +206,7 @@ struct ZMUserSessionBuilder {
         self.coreCryptoProvider = coreCryptoProvider
         self.coreDataStack = coreDataStack
         self.cryptoboxMigrationManager = cryptoboxMigrationManager
-        self.dependencies = buildUserSessionDependencies(coreDataStack: coreDataStack)
+        self.dependencies = buildUserSessionDependencies(coreDataStack: coreDataStack, analyticsSessionConfiguration: analyticsSessionConfiguration)
```

Comment:
Is this still needed?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/e12bacb446dc277c68c1c3efba1ebb121183ea68